[2023-04-09T00:40:14.302+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: api_dag.is_api_available manual__2023-04-09T00:40:11.931406+00:00 [queued]>
[2023-04-09T00:40:14.309+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: api_dag.is_api_available manual__2023-04-09T00:40:11.931406+00:00 [queued]>
[2023-04-09T00:40:14.310+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-04-09T00:40:14.311+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 2
[2023-04-09T00:40:14.312+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-04-09T00:40:14.320+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): is_api_available> on 2023-04-09 00:40:11.931406+00:00
[2023-04-09T00:40:14.328+0000] {standard_task_runner.py:55} INFO - Started process 10096 to run task
[2023-04-09T00:40:14.334+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'api_dag', 'is_api_available', 'manual__2023-04-09T00:40:11.931406+00:00', '--job-id', '83', '--raw', '--subdir', 'DAGS_FOLDER/api.py', '--cfg-path', '/tmp/tmpk9e52l7w']
[2023-04-09T00:40:14.339+0000] {standard_task_runner.py:83} INFO - Job 83: Subtask is_api_available
[2023-04-09T00:40:14.405+0000] {task_command.py:376} INFO - Running <TaskInstance: api_dag.is_api_available manual__2023-04-09T00:40:11.931406+00:00 [running]> on host c12a518d534c
[2023-04-09T00:40:14.459+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=api_dag
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2023-04-09T00:40:11.931406+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-09T00:40:11.931406+00:00
[2023-04-09T00:40:14.461+0000] {http.py:120} INFO - Poking: api/
[2023-04-09T00:40:14.471+0000] {base.py:71} INFO - Using connection ID 'user_api' for task execution.
[2023-04-09T00:40:14.477+0000] {http.py:148} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2023-04-09T00:40:15.098+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2023-04-09T00:40:15.110+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=api_dag, task_id=is_api_available, execution_date=20230409T004011, start_date=20230409T004014, end_date=20230409T004015
[2023-04-09T00:40:15.142+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-04-09T00:40:15.171+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
